packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelController.java

```diff
@@ -140,19 +123,19 @@ ResponseEntity<Model> getModel(@PathVariable("id") UUID id) {
 					try {
 						// Fetch the Document extractions
 						final Optional<DocumentAsset> document = documentAssetService
-								.getDocumentAsset(UUID.fromString(documentId));
+								.getAsset(UUID.fromString(documentId));
 						if (document.isPresent()) {
 							final List<JsonNode> extractions = objectMapper.convertValue(
 									document.get().getMetadata().get("attributes"),
-									new TypeReference<List<JsonNode>>() {
-									});
+                                    new TypeReference<>() {
```

Comment:
what is going on with the tabbing??!?!?!

Reply:
woah. Fixed!